src/js/autocomplete.js

```diff
@@ -189,7 +222,7 @@ export function autocomplete (config) {
 
       dropDown.style.marginLeft = '0'
       dropDown.style.marginTop = element.getBoundingClientRect().height + 'px'
-      this.options = options.map(String)
+      this.options = options;
```

Comment:
I think it is important to keep this code to ensure that when a user has options containing non-string values like `[1,2,3]`, that these numeric values cannot end up being passed as a number into a custom filter function: that would be a breaking change. 

The code needs to be updated of course to only stringify values when not an object.